pandas/tests/extension/base/ops.py

```diff
@@ -263,3 +263,10 @@ def test_unary_ufunc_dunder_equivalence(self, data, ufunc):
         else:
             alt = ufunc(data)
             tm.assert_extension_array_equal(result, alt)
+
+
+class BaseRoundingTests:
+    def test_round(self, data):
+        result = pd.Series(data).round()
```

Comment:
maybe have it only apply to numeric dtypes?

```
if not data.dtype._is_numeric:
    pytest.skip("message here")
```

Reply:
done